### PR TITLE
12.0 partner event

### PR DESCRIPTION
**Pull request by AnilKesariya**
Migrated partner_event module.

### CODE REVIEW

**Review by chienandalu — CHANGES_REQUESTED** [review]
Thanks!

- Review runbot warnings.
- Squash commits a little bit by author/continuity

**Comment by pedrobaeza** [comment]
@AnilKesariya can you please finish the migration?

**Comment by AnilKesariya** [comment]
@pedrobaeza : I've stopped migration because i didn't understand why the first module pull request is not accepted yet. what am I missing in the first module PR? I will continue with other modules once i will close the first PR, after understanding all the checks.

**Comment by pedrobaeza** [comment]
Can you point me to that first one?

**Comment by AnilKesariya** [comment]
@pedrobaeza : here is link for PR
https://github.com/OCA/event/pull/122#issuecomment-431280240
I've created this pull request, May i know what's wrong in that commit?

**Comment by pedrobaeza** [comment]
Ah, but you are talking about this same PR... Reading your words, it seemed to me that there was another one. You have to click on details for seeing the problem on each CI. On Travis, it's only a little flake8 (syntax and guidelines) problem:
```
partner_event/__manifest__.py:17:25: E231 missing whitespace after ':'
```
and another little one for runbot:
```
2018-10-19 06:46:18,233 151 WARNING openerp_test odoo.models: The model res.partner.register.event has no _description
```

**Comment by AnilKesariya** [comment]
@pedrobaeza  : thanks a lot for pointing out the exact error, but I didn't see those info when i click on the detail of failed checks. how to see this info?

**Comment by pedrobaeza** [comment]
For Travis:
travis

For runbot:

runbot

**Comment by AnilKesariya** [comment]
thanks @pedrobaeza  :), This will encourage me more to contribute.

**Comment by pedrobaeza** [comment]
@AnilKesariya you're welcome. You will get used to this very soon, you'll see.

**Comment by AnilKesariya** [comment]
Resolved the failed checks.

File: partner_event/models/event_event.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 # Copyright 2015 Tecnativa S.L. - Javier Iniesta
 # Copyright 2016 Tecnativa S.L. - Antonio Espinosa
 # Copyright 2016 Tecnativa S.L. - Vicent Cubells
+# Copyright 2018 Jupical Technologies Pvt. Ltd. - Anil Kesariya
```
Comment (pedrobaeza) [inline partner_event/models/event_event.py]: You shouldn't put your copyright on files that you haven't touched. Put your name on contributors part on the README, and copyright on the manifest file + touched files.

Comment (AnilKesariya) [reply partner_event/models/event_event.py]: Ok @pedrobaeza, Clear.

**Comment by AnilKesariya** [comment]
@pedrobaeza : I've removed copyright from untouched file.

**Review by pedrobaeza — APPROVED** [review]
@AnilKesariya I have removed your addition to authors, as the little changes you have made don't deserve the co-authorship of the module. Contributors part on the README continues the same, as this is the place to put new credits. You promote to co-author when add a significant part of code or features.

**Comment by pedrobaeza** [comment]
@chienandalu can you finish your review?

@AnilKesariya I have squashed a bit commit history

File: partner_event/README.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/event/issues>`_.
+In case of trouble, please check there if your issue has already been
+reported. If you spotted it first, help us smashing it by providing a detailed
+and welcomed feedback.
+
+Known Issues / Roadmap
+======================
+
+* In registration contact field has no onchange check so changing the partner
```
Comment (chienandalu) [inline partner_event/README.rst]: Remove this, as it's no longer true.

**Review by chienandalu — CHANGES_REQUESTED** [review]
Tested in runbot :+1: 

Minor comment below.

**Review by cristinamartinrod — APPROVED** [review]
Tested on runbot. LGTM 👍🏻

**Pull request merged**